### ISSUE TITLE
Rule Proposal: Warn if "is" Protractor/WebDriverJS methods used inside the if conditions

### ISSUE THREAD

**Issue by alecxe** [alecxe/eslint-plugin-protractor#28]
Example:

``` js
if (elm.isDisplayed()) {
    // ...
}
```

The problem is - `isDisplayed()` returns _promise_ which itself is always truthy regardless of the real displayedness state. The promise has to be resolved to obtain the real true/false value.

We, of course, cannot cover all the possible problem variations, but can work with simple cases like this. Methods we should support:
- `isDisplayed()`
- `isEnabled()`
- `isSelected()`
- ...

**Comment by alecxe** [comment]
Closed by f9f9c79be2d8e8e980884b755cf6a2a8029c058c.
